src/dotnet/commands/dotnet-new/Program.cs

```diff
@@ -140,7 +140,8 @@ public static int Run(string[] args)
                                    new { Name = "Web", isMsBuild = true }, 
                                    new { Name = "Lib", isMsBuild = true },
                                    new { Name = "Mstest", isMsBuild = true },
-                                   new { Name = "Xunittest", isMsBuild = true }
+                                   new { Name = "Xunittest", isMsBuild = true },
```

Comment:
@livarcocc tested it out, and nope. The logic is such that it checks if the file with the same name (e.g. `Program.cs`) exists in the folder and errors out on that.